app/assets/javascripts/pageflow/ui/views/inputs/check_box_group_input_view.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Input view for attributes storing configuration hashes with boolean values.
+ *
+ * @see {@link module:pageflow/ui.pageflow.inputView pageflow.inputView} for further options
+ * @class
+ * @memberof module:pageflow/ui
+ */
+pageflow.CheckBoxGroupInputView = Backbone.Marionette.ItemView.extend({
+  mixins: [pageflow.inputView],
+
+  template: 'pageflow/ui/templates/inputs/check_box_group_input',
+  className: 'check_box_group_input',
+
+  events: {
+    'change': 'save'
+  },
+
+  ui: {
+    label: 'label',
+    container: '.check_boxes_container'
+  },
+
+  initialize: function() {
+    if (!this.options.texts) {
+      if (!this.options.translationKeys) {
+        var translationKeyPrefix = this.options.translationKeyPrefix ||
+          pageflow.i18nUtils.findKeyWithTranslation(this.attributeTranslationKeys('values', {
+            fallbackPrefix: 'activerecord.values'
+          }));
+
+        this.options.translationKeys = _.map(this.options.values, function(value) {
+          return translationKeyPrefix + '.' + value;
+        }, this);
+      }
+
+      this.options.texts = _.map(this.options.translationKeys, function(key) {
+        return I18n.t(key);
+      });
+    }
+  },
+
+  onRender: function() {
+    this.ui.label.attr('for', this.cid);
+    this.appendOptions();
+    this.load();
+    this.listenTo(this.model, 'change:' + this.options.propertyName, this.load);
+  },
+
+  appendOptions: function () {
+    _.each(this.options.values, function(value, index) {
+      var option = '<div class="check_box">'+
+        '<input type="checkbox" name="'+value+'" />'+
+        '<label>'+this.options.texts[index]+'</label></div>';
```

Comment:
If we move the input into the label, clicking the label will allow toggling the input.